src/pages/staticPages/datasetView/DatasetPage.tsx

```diff
@@ -20,12 +19,11 @@ export class DatasetPageStore {
         },
     });
 
-    readonly downloadList = remoteData(() => {
-        if (getServerConfig().app_name === 'public-portal') {
```

Comment:
before we were only doing the HTTP request on public portal, maybe it's good to keep it that way? I.e. maybe we can do something like when `study_download_url` is not defined, we don't even fire the request?

Reply:
Will do!!